sdk/react/panels/videoView.js

```diff
@@ -121,10 +121,13 @@ class VideoView extends React.Component {
     }
   };
 
-  onSeekPressed = (isForward) => {
-    let configSeekValue = (isForward) ? this.props.config.skipControls.skipForwardTime : this.props.config.skipControls.skipBackwardTime;
-    configSeekValue = Utils.restrictSeekValueIfNeeded(configSeekValue);
-    const seekValue = (isForward) ? configSeekValue : configSeekValue * (-1);
+  onSeekPressed = (skipCountValue) => {
+    if (skipCountValue == 0) { return null; }
+
+    let configSeekValue = (skipCountValue > 0) ? this.props.config.skipControls.skipForwardTime : this.props.config.skipControls.skipBackwardTime;
+    //configSeekValue = Utils.restrictSeekValueIfNeeded(configSeekValue);
```

Comment:
why do we remove this? We need it so that we accept values between 1-99 only.

Reply:
It's a temporal solution because in the ticket we discussed another behavior.

Without this PR we can skip only in interval of 1..99, but if we are to sum up all taps on skip button it's possible that we will get a value larger than 99. 

Still we have a check further that it's still in asset duration.